plone-5/Makefile

```diff
@@ -7,6 +7,11 @@ all: build
 build:
 	docker build --build-arg POSTGRES_PASSWORD=$(POSTGRES_PASSWORD) --tag plone .
```

Comment:
not sure if its documented but this will only work if the env var POSTGRES_PASSWORD is set, its possible we want toset it, or maybe stick a conditional in to tell people off for not doing so

```
if [ -z "$POSTGRES_PASSWORD" ]; then
  echo "POSTGRES_PASSWORD is not set"
  exit
fi
```

something like that...maybe... my bash ain't that great tbh.